scss/detail.scss

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

Comment:
You can move this import to the `application.scss` file

Reply:
done